pkg/ibmvpc/parser.go

```diff
@@ -675,10 +690,11 @@ func getTgwObjects(c *datamodel.ResourcesContainerModel,
 		if _, ok := tgwToSkip[tgwUID]; ok {
 			continue
 		}
+		tgw, ok := tgwIDToTgw[tgwUID]
```

Comment:
```suggestion
		tgw, hasTgwConfig:= tgwIDToTgw[tgwUID]
```
Better use meaningful name here .
Another option is to have just `tgw = tgwIDToTgw[tgwUID]` , and check if `tgw` is `nil` to check if it has no config

Reply:
also, better rename `tgw` to `tgwFromConfig`  (line 732 defines another var named `tgw`, and this may be confusing)